stories/SearchBox.stories.js

```diff
@@ -14,7 +14,10 @@ stories
       hasPlayground={true}
       linkedStoryGroup="SearchBox.stories.js"
     >
-      <SearchBox showLoadingIndicator={boolean('showLoadingIndicator', true)} />
+      <SearchBox
+        inputId="search-box"
```

Comment:
not needed in the storybook I think (although of course a fun way to try it out). Could you also remove it from the other PRs?

Reply:
All PRs have been updated to remove these story updates!